join.md

```diff
@@ -0,0 +1,292 @@
+way around.
+
+
+Header and icon illustration source:
+https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?type=design&node-id=3520%3A5225&mode=design&t=NTwtjgAcEcRcLbcb-1
+
+-->
+
+{% include picture.html
+   image = "/assets/images/join/contribute.jpg"
+   retina = "/assets/images/join/contribute@2x.jpg"
+   mobile = "/assets/images/join/contribute-mobile.jpg"
+   mobileRetina = "/assets/images/join/contribute-mobile@2x.jpg"
+   alt-text = "Pencils and markers"
+   width = 1600
+   height = 900
+   layout = "full-width"
+%}
+
+# Join Bitcoin Design
+
+**So you’re interested in joining? Fantastic! We welcome, appreciate and encourage newcomers and contributions.**
+
+To set you up for success, we’ve created this guide to help. It will help you set a solid foundation for yourself by navigating you through the basics of contributing to open-source projects in this community. In it, you’ll find resources for many of our documents and materials that you’ll need as you embark on your journey.
+
+You’re welcome to ask questions as you go along. In fact, please do. Sometimes you might even highlight a question that others have pondered too.
+
+We structured this guide in milestones that you can go through one by one at your own pace.
+
+## Milestone 1: Say hi
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/join/say-hi.jpg"
+   retina = "/assets/images/join/say-hi@2x.jpg"
+   alt-text = ""
+   width = 150
+   height = 150
+   layout = "float-left"
+%}
+
+It’s time to join the [community on Discord](https://discord.gg/K7aQ5PErht) and say hi in the [#introductions](https://discord.com/channels/903125802726596648/903126234232422411) channel. You can use the format below:
+
+</div>
+
+- 👋 A little about me
+- 🥕 What I am currently working on?
+- 🚀 What are my interest?
+- 🤩 A random question for the community?
+- 🥁 A design joke that makes you laugh?
+
+Then go ahead and browse the various channels to see what projects are being discussed and chime in. Open-source relies on proactive contributions, and participating in conversations is one of the most essential ways to do so.
+
+To stay up-to-date:
+- Subscribe to the [newsletter]({{ '/newsletter/' | relative_url }})
+- Follow on [social media]({{ '/social-media/' | relative_url }}), like [Nostr](https://nosta.me/_@bitcoin.design)
+- Subscribe to the [community calendar]({{ '/calendar/' | relative_url }})
+
+## Milestone 2: Explore
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/join/explore.jpg"
+   retina = "/assets/images/join/explore@2x.jpg"
+   alt-text = ""
+   width = 150
+   height = 150
+   layout = "float-left"
+%}
+
+Now it’s time to learn more about all the projects and collaborations that happen in and around the community. Projects are incredibly diverse, so it may take a little bit of time to get a feel for what’s out there and what you connect with well.
```

Comment:
```suggestion
Now it’s time to learn more about all the projects and collaborations that happen in and around the community. Projects are incredibly diverse and are at varying levels of progress, so it may take a little bit of time to get a feel for what’s out there and what you connect with well.
```